IOHexperimenter/python.md

```diff
@@ -15,30 +19,58 @@ pip install ioh
 
 ## Usage
 
-The following walks through the typical use cases of the IOHexperimenter. 
-You can find this in a jupyter notebook [on this page]() to run it in an interactive manner.
+There are a few main ways of using the IOHexperimenter. All of them are based on the 'function' as defined by IOHexperimenter. 
+Then, one can either use this function directly, or, using the 'experimenter'-class, automate the entire benchmarking pipeline.
+On this page, we will first introduce the basic aspects of this 'function' object, which will be the part of the codebase which your algorithm needs to interact with. Then, we introduce several usecases of IOHexperimenter, ranging from running an algorithm on one existing problem to complete benchmarking pipelines.
 
 ### Create a function object
 The structure of the IOHexperimenter in python is almost equivalent to the C++ version, but with a few ease-of-use features added, such as easy access to any existing benchmark problem usin the 'get_problem' function:
 ```python
 # Import the get_problem function
 from ioh import get_problem
 ```
-To check the usage and parameterization of this (and most other) functionality, we provide built-in docstrings, acessible as usual:
+To check the usage and parameterization of this (and most other) functionality, we provide built-in docstrings. These are available in Ipython as follows:
 
 ```python
 #View docstring of get_problem
 ?get_problem
```

Comment:
Should we remove the ?

Reply:
I think we should keep them so we can have the tutorial match the notebook example